### PR TITLE
Update plugins.md

### PR DESCRIPTION
**Pull request by mocnik-science**

### CODE REVIEW

**Comment by ghybs** [comment]
Hi @mocnik-science,

I would be happy to merge your PR if you could resolve the merge conflicts please?

**Comment by mocnik-science** [comment]
done!

File: docs/plugins.md [hunk 1]
```diff
@@ -2101,11 +2101,11 @@ Powerful multi-purpose libraries for data visualization.
 <table class="plugins"><tr><th>Plugin</th><th>Description</th><th>Maintainer</th></tr>
 	<tr>
 		<td>
-			<a href="https://github.com/giscience/geogrid.js">geogrid.js</a>
+			<a href="https://github.com/mocnik-science/geogrid.js">geogrid.js</a>
 		</td><td>
-			Displays data aggregated by the ISEA3H discrete global grid system.  The data can, e.g., be delivered by using <a href="https://github.com/giscience/measures-rest">Measures REST</a> (a framework to deliver data aggregated by the grid) or <a href="https://github.com/giscience/geogrid">geogrid</a> (a library for handling the grid in case that you want to aggregate data manually).
+			Displays data aggregated by the ISEA3H discrete global grid system.  The data can, e.g., be delived by using <a href="https://github.com/mocnik-science/geogrid">geogrid</a> (a library for handling the grid in case that you want to aggregate data manually) or <a href="https://github.com/giscience/measures-rest">Measures REST</a> (a framework to deliver data aggregated by the grid).
```
Comment (ghybs) [inline docs/plugins.md]: typo "delivered" see #6812

**Comment by mocnik-science** [comment]
see new pull request: #6856

**Pull request closed without merging**